R/eta_squared.R

```diff
@@ -80,22 +80,31 @@ eta_squared.anova <- eta_squared.aov
 
 
 #' @export
-eta_squared.aovlist <- function(model, partial = TRUE, ci = NULL, iterations = 1000) {
-  stop("Eta squared not implemented yet for repeated-measures ANOVAs.")
-
-  # params <- .extract_parameters_anova(model)
-  # values <- .values_aov(params)
-  #
-  # if (!"Residuals" %in% params$Parameter) {
-  #   stop("No residuals data found. Omega squared can only be computed for simple `aov` models.")
-  # }
-  #
-  # mapply(function(p, v) {
-  #   .extract_eta_squared(p, v, partial)
-  # }, split(params, params$Group), values, SIMPLIFY = FALSE)
+eta_squared.aovlist <- function(model, partial = TRUE, ci = NULL) {
+  if (isFALSE(partial))
```

Comment:
Is this a new trend that you and Daniel recently adopted - not putting brackets? As far as I know, I thought *I* was the only one pythonista here :)